test/utils.ts

```diff
@@ -1,5 +1,7 @@
-import crypto from 'crypto';
+import { BigNumber, Wallet } from 'ethers';
 
-export const createRandomAddress = () =>
-  `0x${crypto.randomBytes(20).toString('hex')}`;
-export const createRandomeValue = (value: number) => `${Math.random() * value}`;
+export const createRandomAddress = () => Wallet.createRandom().address;
+export const createRandomValue = (value: number) =>
+  `${(Math.random() * value).toFixed(0)}`;
+export const createRandomBigNumber = (base: number) =>
```

Comment:
Let's use the same argument name in these functions.

Reply:
updated